modules/npmscripts.config.js

```diff
@@ -25,7 +25,402 @@ const CHECK_AND_FIX_GLOBS = [
+				'@babel/helper-module-transforms': true,
+				'@babel/helper-optimise-call-expression': true,
+				'@babel/helper-plugin-utils': true,
+				'@babel/helper-regex': true,
+				'@babel/helper-remap-async-to-generator': true,
+				'@babel/helper-replace-supers': true,
+				'@babel/helper-simple-access': true,
+				'@babel/helper-split-export-declaration': true,
+				'@babel/helper-wrap-function': true,
+				'@babel/highlight': true,
+				'@babel/parser': true,
+				'@babel/plugin-proposal-async-generator-functions': true,
+				'@babel/plugin-proposal-json-strings': true,
+				'@babel/plugin-proposal-object-rest-spread': true,
+				'@babel/plugin-proposal-optional-catch-binding': true,
+				'@babel/plugin-proposal-unicode-property-regex': true,
+				'@babel/plugin-syntax-async-generators': true,
+				'@babel/plugin-syntax-json-strings': true,
+				'@babel/plugin-syntax-object-rest-spread': true,
+				'@babel/plugin-syntax-optional-catch-binding': true,
+				'@babel/plugin-transform-arrow-functions': true,
+				'@babel/plugin-transform-async-to-generator': true,
+				'@babel/plugin-transform-block-scoped-functions': true,
+				'@babel/plugin-transform-block-scoping': true,
+				'@babel/plugin-transform-classes': true,
+				'@babel/plugin-transform-computed-properties': true,
+				'@babel/plugin-transform-destructuring': true,
+				'@babel/plugin-transform-dotall-regex': true,
+				'@babel/plugin-transform-duplicate-keys': true,
+				'@babel/plugin-transform-exponentiation-operator': true,
+				'@babel/plugin-transform-for-of': true,
+				'@babel/plugin-transform-function-name': true,
+				'@babel/plugin-transform-literals': true,
+				'@babel/plugin-transform-modules-amd': true,
+				'@babel/plugin-transform-modules-commonjs': true,
+				'@babel/plugin-transform-modules-systemjs': true,
+				'@babel/plugin-transform-modules-umd': true,
+				'@babel/plugin-transform-named-capturing-groups-regex': true,
+				'@babel/plugin-transform-new-target': true,
+				'@babel/plugin-transform-object-super': true,
+				'@babel/plugin-transform-parameters': true,
+				'@babel/plugin-transform-regenerator': true,
+				'@babel/plugin-transform-shorthand-properties': true,
+				'@babel/plugin-transform-spread': true,
+				'@babel/plugin-transform-sticky-regex': true,
+				'@babel/plugin-transform-template-literals': true,
+				'@babel/plugin-transform-typeof-symbol': true,
+				'@babel/plugin-transform-unicode-regex': true,
+				'@babel/preset-env': true,
+				'@babel/template': true,
+				'@babel/traverse': true,
+				'@babel/types': true,
+				'ansi-styles': true,
+				bootstrap: true,
+				browserslist: true,
+				'caniuse-lite': true,
+				chalk: true,
+				commander: true,
+				'electron-to-chromium': true,
+				lodash: true,
+				'regenerate-unicode-properties': true,
+				'regenerator-transform': true,
+				'regexp-tree': true,
+				rw: true,
+				semver: true,
+				'source-map': true,
+			},
+		},
+	},
+	federation: {
+		mode: 'disabled',
```

Comment:
Hm, so I guess this answers my question above... the "default" in liferay-portal is effectively going to be `'disabled'` for a while (until everything is moved to use federation; that seems like it will still be a fair bit of work, right? Or are you thinking that the first ones were the hard ones and the rest will be easier from here on?)

`'default'` not being the default is a bit counterintuitive, so we might get some pushback from Brian on this unless we can make a confident argument that it will be short-lived.

Reply:
We'll see what happens... :roll_eyes: